components/openzwave-adapter/src/lib.rs

```diff
@@ -490,11 +490,11 @@ impl taxonomy::adapter::Adapter for OpenzwaveAdapter {
         }).collect()
     }
 
-    fn register_watch(&self, mut values: Vec<(TaxoId<Channel>, Option<Value>, Box<ExtSender<WatchEvent>>)>) -> Vec<(TaxoId<Channel>, Result<Box<AdapterWatchGuard>, TaxoError>)> {
+    fn register_watch(&self, mut values: Vec<(TaxoId<Channel>, Option<Value>, Box<ExtSender<WatchEvent<Value>>>)>) -> Vec<(TaxoId<Channel>, Result<Box<AdapterWatchGuard>, TaxoError>)> {
```

Comment:
Unfortunately, an alias wouldn't work. If you do `type Foo = WatchEvent<Value>`, you cannot use `Foo::Enter` and `Foo:Exit`, so this would make things more complicated.

Also, there are already a few uses of `WatchEvent<T>` where `T` is not `Value` (well, only one at the moment). Defining two different types with the same structure didn't seem like a good use of Rust's ADT.